### ISSUE TITLE
SVG Badge Image

### ISSUE THREAD

**Issue by kayaelle** [OpenWorksGroup/Open-BadgeBot#2]
This task is for the badge image (unearned)

See notes in wiki:
https://github.com/OpenWorksGroup/Open-BadgeBot/wiki/SVG

Added svg_file to badge-participate-community.js This can reference an svg file.

Need some brainstorming...

**Comment by zwallacedev** [comment]
Proposal for how we can handle svg/pngs in our project for managing the assets. I would like to put a folder structure in that reflects:
- public
-- images
--- png
--- svg

This will allow us to have a few different structural wins in terms of the ability for a new person to read the files. What are your thoughts on this one? @kayaelle

**Comment by zwallacedev** [comment]
Btw -- I'm going to go with this for the PR, but I just wanted to give a heads up that I was thinking about it this way.

**Comment by kayaelle** [comment]
Hey @zwallacedev ! Let's talk about it when we meet up. Today I need to figure out how we're going to store everything that's needed for an earned badge.

Thinking about what can be generated on the fly and what needs to be stored. For instance, in this badge, can we make the Earner png as we need it? 

I'm going to look into using gists for badge json data. 

https://github.com/jonschlinkert/gist
https://hackernoon.com/3-ways-to-get-data-from-github-gists-7e1f73b9db9d

**Comment by zwallacedev** [comment]
Ah, that's interesting. We can talk when we meet up, but it will be possible to create a png on the fly. The tricky part is going to be changing the SVG based on criteria. If we store the SVGs, we may not ever need to store the PNGs. As I research this more, it seems like my initial plan may not be perfect, but I do think we should have some sort of `images` folder that will contain all of the images we will need for the badges. This could be sorted by badges, so we could have specific SVGs within the folder structure like:
```
- public
--images
---svg
----badge_summit
-----first_badge
-----second_badge
-----third_badge
```
This would be a temporary solution until we determine how to programmatically split the SVG.

**Comment by zwallacedev** [comment]
This was closed in PR #18
